### PR TITLE
Bump Synpress Version

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR bumps all testing tools versions to the latest available. This will resolve some vulnerability issues.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: dapp-example/tests/e2e/specs/index.spec.js [hunk 1]
```diff
@@ -85,6 +85,12 @@ describe('Test Core Hedera User Scenarios', function() {
   });
 
   function mandatoryTests() {
+
+    beforeEach(() => {
```
Comment (Nana-EC) [inline dapp-example/tests/e2e/specs/index.spec.js:89]: Q: What's this doing and how come it's needed?

**Review by Nana-EC — COMMENTED** [review]
LG.
Curious of the connect account addition. What's the context

File: dapp-example/tests/e2e/specs/index.spec.js (hunk 1 above)
Comment (georgi-l95) [reply dapp-example/tests/e2e/specs/index.spec.js:89]: Yes, for some reason it clears the context of the page after each test with the latest versions of the plugin. So we need to visit the page before each tests.
There are like hundreds of changes from version 3.0.5 to 3.7.1 and most of them are breaking, so I'm not sure where this behaviour was introduced.

Comment (Nana-EC) [reply dapp-example/tests/e2e/specs/index.spec.js:89]: Okay any comments would be valuable. Not sure of the operation but seems like it's saying before every tests navigate to home and if there's a "Connect Account" item then click it

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**